src/main/java/core/basesyntax/EmploeesSalary.java

```diff
@@ -0,0 +1,22 @@
+package core.basesyntax;
+
+public class EmploeesSalary {
```

Comment:
```suggestion
public class Employee {
```
do you really need two model classes?